sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/contacts/ExposureNewEntryPage.java

```diff
@@ -0,0 +1,60 @@
+package org.sormas.e2etests.pages.application.contacts;
+
+import org.openqa.selenium.By;
+
+public class ExposureNewEntryPage {
+  public static final By EXPOSURE_DETAILS_KNOWN =
```

Comment:
Please name your selectors related to the items which are mapped.

Take a look to the rest of the classes which are used to mapping to see the pattern used.

We are using:
_COMBOBOX for comboboxes
_DATE for dates
_INPUT for textboxes where data can be inserted
_TEXT or _VALUE or _LABEL for labels where data are stored

Reply:
addressed